canonical-web-and-design/anbox-cloud.io
# Add new cookie module to site

## Done

- Add the new cookie module to the site

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See the new cookie popup and try the settings
- Test the `Manage your tracker settings` link from the footer

## Issue / Card

Fixes #119
